### DISCUSSION TITLE
About getting a "pro" organization

### DISCUSSION THREAD

**Discussion by greatwallet** [community/community#22737] [New to GitHub]
<p>I am a “pro”-tagged user and created a team on github. All collaborators in the team are all “pro” users. (Actually we are all students who’ve applied for students pack). Can we update the github team to “pro” free automatically, or do we have to pay as any other “free” users?</p>
<p>Thanks!</p>

**Comment by wordsoup** [comment, marked as answer]
<p>Your GitHub Student Developer Pack benefits do not extend to other accounts, such as organizations. Students are also not eligible for academic discounts on organizations for a Team subscription. However, mentors of FIRST Robotics teams can apply for an organizational discount on behalf of their students, as well as teachers.</p>
<p>Your teacher can <a href="https://docs.github.com/en/github/teaching-and-learning-with-github-education/applying-for-an-educator-or-researcher-discount#upgrading-your-organization" rel="nofollow noopener">apply</a> for an organizational discount on your behalf for your team to collaborate on projects.</p>
<p>Instead of an organizational discount, you can also <a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/inviting-collaborators-to-a-personal-repository" rel="nofollow noopener">invite</a> your teammates to collaborate within repositories on your personal account.</p>
